neuralprophet/time_dataset.py

```diff
@@ -158,6 +159,32 @@ def init_after_tabularized(self, inputs, targets=None):
                     sample[key] = data[index]
             self.samples.append(sample)
 
+    def filter_samples_after_init(
+        self,
+        forecast_frequency=None,
+    ):
+        """Filters samples from the dataset based on the forecast frequency.
+
+        Parameters
+        ----------
+            forecast_frequency : int
+                periodic interval in which forecasts should be made.
+
+            Note
+            ----
+            E.g. if forecast_frequency=7, forecasts are only made on every 7th step (once in a week in case of daily resolution).
+        """
+        if forecast_frequency is None:
```

Comment:
You could also add the case of `forecast_frequency==1` here

Reply:
good catch, thanks!